pipeline_noted.ipynb

```diff
@@ -358,7 +387,12 @@
+    "The first argument of `load_videos` should be the path that contains the videos, which should be `dpath` we already defined. The argument `pattern` is optional and is the [regular expression](https://docs.python.org/3/library/re.html) used to filter files under the specified folder. The default value **'msCam[0-9]+\\.avi$'** means that a file can only be loaded if its filename contain **'msCam'** then followed by at least one digit of number then **'.avi'** as the end of the filename. This can be changed to suit the naming convention of your videos. `in_memory` specify whether the whole video should be loaded into memory. `dtype` is the underlying [data types](https://docs.scipy.org/doc/numpy-1.15.0/user/basics.types.html) of the data. Usually `float32` is good enough and should be preferred to save memory demand. The resulting \"video array\" `varr` contains three dimensions: `height`, `width`, and `frame`. If you wish to downsample the video, pass in a `dictionary` to `downsample`, whose keys should be the name of dimensions and values an integer specifying how many times that dimension should be reduced. For example, `downsample=dict('frame'=2)` will temporally downsample the video with a factor of 2. Instead, if you do not wish to downsample your data, simply pass in `downsample=None`. `downsample_strategy` will assume two values: either `'subset'`, meaning downsampling are carried out simply by subsetting the data, or `'mean'`, meaning a mean will be calculated on the window of downsampling.\n",
+    "\n",
+    "`param_load_videos` should be specified as python [dictionary](https://docs.python.org/3/tutorial/datastructures.html#dictionaries), whose `keys` are the dimension along which subsetting should be done, and `values` specify how subsetting should be done. **If you do want to modify the parameters, you can either go back to the initial parameter setting and change it there. Or by recalling the parameter here, then modify it by changing the values in this specific parameter dictionary.** For example, if you want to stop downsampling your data, you can type in the console: \n",
```

Comment:
I don't think we have properly introduced consoles, and it's a pain to attach a console to a running kernel in traditional jupyter notebook (not jupyterlab)